### PR TITLE
feat(package.json): Update package name

### PR DESCRIPTION
**Pull request by yehor-manzhula**
## Description
Update package name, readme file

## Motivation and Context
After fork package name needs to be update

## How Has This Been Tested?
Tests are green, build is green

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,18 @@
...
 
-[![NPM](https://nodei.co/npm/inversify-express-utils.png?downloads=true&downloadRank=true)](https://nodei.co/npm/inversify-express-utils/)
-[![NPM](https://nodei.co/npm-dl/inversify-express-utils.png?months=9&height=3)](https://nodei.co/npm/inversify-express-utils/)
 
 Some utilities for the development of express applications with Inversify.
 
 ## Installation
 
-You can install `inversify-express-utils` using npm:
+You can install `express-inversify` using npm:
```
Comment (b1ff) [inline README.md:9]:
```suggestion
You can install `@namecheap/express-inversify` using npm:
```

File: README.md [hunk 2]
```diff
@@ -1,28 +1,18 @@
...
 
 ```sh
-npm install inversify inversify-express-utils reflect-metadata --save
+npm install inversify express-inversify reflect-metadata --save
```
Comment (b1ff) [inline README.md]:
```suggestion
npm install inversify @namecheap/express-inversify reflect-metadata --save
```

File: README.md [hunk 3]
```diff
@@ -35,7 +25,7 @@ The following example will declare a controller that responds to `GET /foo'.
 
 ```ts
 import * as express from "express";
-import { interfaces, controller, httpGet, httpPost, httpDelete, request, queryParam, response, requestParam } from "inversify-express-utils";
+import { interfaces, controller, httpGet, httpPost, httpDelete, request, queryParam, response, requestParam } from "express-inversify";
```
Comment (b1ff) [inline README.md]:
```suggestion
import { interfaces, controller, httpGet, httpPost, httpDelete, request, queryParam, response, requestParam } from "@namecheap/express-inversify";
```

File: README.md [hunk 4]
```diff
@@ -82,13 +72,13 @@ Then, pass the container to the InversifyExpressServer constructor. This will al
...
 In order for the InversifyExpressServer to find your controllers, you must bind them to the `TYPE.Controller` service identifier and tag the binding with the controller's name.
-The `Controller` interface exported by inversify-express-utils is empty and solely for convenience, so feel free to implement your own if you want.
+The `Controller` interface exported by express-inversify is empty and solely for convenience, so feel free to implement your own if you want.
 
 ```ts
 import * as bodyParser from 'body-parser';
 
 import { Container } from 'inversify';
-import { interfaces, InversifyExpressServer, TYPE } from 'inversify-express-utils';
+import { interfaces, InversifyExpressServer, TYPE } from 'express-inversify';
```
Comment (b1ff) [inline README.md]:
```suggestion
import { interfaces, InversifyExpressServer, TYPE } from '@namecheap/express-inversify';
```

File: README.md [hunk 5]
```diff
@@ -122,7 +112,7 @@ Since the `inversify-express-util@5.0.0` release. The `@injectable` annotation i
 If you run the application multiple times within a shared runtime process (e.g. unit testing) you might need to clean up the existing metadata before each test.
 
 ```ts
-import { cleanUpMetadata } from "inversify-express-utils";
+import { cleanUpMetadata } from "express-inversify";
```
Comment (b1ff) [inline README.md]:
```suggestion
import { cleanUpMetadata } from "@namecheap/express-inversify";
```

File: README.md [hunk 6]
```diff
@@ -287,7 +274,7 @@ The benefit of the latter two methods is that since your controller is no longer
 import { injectable, inject } from "inversify";
 import {
     controller, httpGet, BaseHttpController, HttpResponseMessage, StringContent
-} from "inversify-express-utils";
+} from "express-inversify";
```
Comment (b1ff) [inline README.md]:
```suggestion
} from "@namecheap/express-inversify";
```

File: README.md [hunk 7]
```diff
@@ -319,7 +306,7 @@ On the BaseHttpController, we provide a litany of helper methods to ease returni
 import { injectable, inject } from "inversify";
 import {
     controller, httpGet, BaseHttpController
-} from "inversify-express-utils";
+} from "express-inversify";
```
Comment (b1ff) [inline README.md]:
```suggestion
} from "@namecheap/express-inversify";
```

**Review by b1ff — COMMENTED** [review]
please fix references to packages - add @namecheap scope

**Comment by StyleT** [comment]
Here we should switch license to `Apache-2.0` and add `NOTICE.md` file, see examples:
- https://github.com/namecheap/ilc/blob/master/NOTICE.md
- https://github.com/namecheap/ilc/blob/master/LICENSE.txt

**Review by b1ff — APPROVED** [review]

File: package.json [hunk 8]
```diff
@@ -31,9 +31,9 @@
   "author": "Cody Simms",
   "license": "MIT",
```
Comment (StyleT) [inline package.json:32]: Pls correct version here too

File: package.json [hunk 9]
```diff
@@ -31,9 +31,9 @@
   "author": "Cody Simms",
```
Comment (StyleT) [inline package.json:31]: Namecheap, Inc

**Review by StyleT — CHANGES_REQUESTED** [review]

**Review by StyleT — APPROVED** [review]

**Pull request merged**